### ISSUE TITLE
ApplicationInsights:Failed to propagate context in Azure Functions [

### ISSUE THREAD

**Issue by paulsmithkc** [Azure/azure-functions-nodejs-library#51]
#### Investigative information

Please provide the following:

- Timestamp: Between 16 Jan, 12:14 PM CST (2023-01-16T20:14:00Z) and 17 Jan 12:57 CST (2023-01-17T20:57:00Z)
- Invocation ID: 32fd51cf-df2e-4920-8eca-25b7abf15ad4
- Region: West US 2

#### Repro steps

Provide the steps required to reproduce the problem:

1. Create and deploy a function app
2. In an http trigger attempt to log a JSON object as the message:

```
const healthTrigger: AzureFunction = async function(context: Context) {
  context.log.info('{"method":"healthTrigger","message":"triggered"}');
  context.res = { body: 'OK', status: 200 };
};
```

#### Expected behavior

Message is logged to AI and no errors are generated.

#### Actual behavior

The following error is written to the console:

```
ApplicationInsights:Failed to propagate context in Azure Functions [
  at AzureFunctionsHook.<anonymous> (D:\home\site\wwwroot\node_modules\[redacted]\node_modules\applicationinsights\out\AutoCollection\AzureFunctionsHook.js:95:38)
  at AzureFunctionsHook._propagateContext (D:\home\site\wwwroot\node_modules\[redacted]\node_modules\applicationinsights\out\AutoCollection\AzureFunctionsHook.js:87:16)
  at AzureFunctionsHook._propagateContext (D:\home\site\wwwroot\node_modules\[redacted]\node_modules\applicationinsights\out\AutoCollection\AzureFunctionsHook.js:87:16)
  at step (D:\home\site\wwwroot\node_modules\[redacted]\node_modules\applicationinsights\out\AutoCollection\AzureFunctionsHook.js:33:23)
  at Object.next (D:\home\site\wwwroot\node_modules\[redacted]\node_modules\applicationinsights\out\AutoCollection\AzureFunctionsHook.js:14:53)
  at new Promise (<anonymous>)
  at __awaiter (D:\home\site\wwwroot\node_modules\[redacted]\node_modules\applicationinsights\out\AutoCollection\AzureFunctionsHook.js:4:12)
  at t.InvocationModel.<anonymous> (D:\Program Files (x86)\SiteExtensions\Functions\4.14.0\workers\node\dist\src\worker-bundle.js:2:51947)
  at new Promise (<anonymous>)
  at y.<anonymous> (D:\Program Files (x86)\SiteExtensions\Functions\4.14.0\workers\node\dist\src\worker-bundle.js:2:34778)
]
```

#### Related information 

Provide any related information 

* Programming language used: Typescript

**Comment by ejizba** [comment]
Hi @paulsmithkc this is actually a bug in the applicationinsights npm module introduced in v2.4.0. I recommend you downgrade the npm package to v2.3.6 as a workaround and track https://github.com/microsoft/ApplicationInsights-node.js/issues/1063 for further updates

**Comment by ejizba** [comment]
Closing as the applicationinsights package released v2.4.1 a few days ago to address this bug
